aws_datalake/modules/emr/variables.tf

```diff
@@ -95,6 +95,12 @@ variable "task_instance_max_count" {
   default     = "4"
 }
 
+variable "emr_cluster_version" {
+  description = "Version of emr cluster"
+  type        = string
+  default     = "6.55"
```

Comment:
Version numbers are semver (x.x.x).  I don't see a version `6.55` in [Amazon's listing](https://docs.aws.amazon.com/emr/latest/ReleaseGuide/emr-release-6x.html).  I see `6.15.0` as the latest.

Reply:
oh my bad